examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -10130,17 +10124,13 @@ class NetworkCommissioningUpdateThreadNetwork : public ModelCommand
 
     CHIP_ERROR SendCommand(ChipDevice * device, uint8_t endpointId) override
     {
-        uint8_t opDataset[chip::Thread::kSizeOperationalDataset];
-        uint32_t opDatasetLen;
-
         ChipLogProgress(chipTool, "Sending cluster (0x0031) command (0x08) on endpoint %" PRIu16, endpointId);
 
         chip::Controller::NetworkCommissioningCluster cluster;
         cluster.Associate(device, endpointId);
-        chip::ArgParser::ParseHexString(mThreadOpDatasetArg, static_cast<uint32_t>(strlen(mThreadOpDatasetArg)), opDataset,
```

Comment:
I did not modify these. I only regenerated the files. I wonder if someone pushed some code to ToT modifying this file not knowing it was autogenerated..